modules/javafx.graphics/src/main/native-glass/mac/a11y/AccessibleBase.m

```diff
@@ -97,106 +96,74 @@ - (void)dealloc
     [super dealloc];
 }
 
-- (jobject)getJAccessible
-{
+- (jobject)getJAccessible {
```

Comment:
should this K&R brace be reverted to the _objectively better_ style used elsewhere in this file?